files/image_config/interfaces/interfaces.j2

```diff
@@ -22,20 +22,20 @@ iface lo {{ 'inet' if prefix | ipv4 else 'inet6' }} static
 auto eth0
 {% if MGMT_INTERFACE %}
 {% for (name, prefix) in MGMT_INTERFACE %}
-iface eth0 {{ 'inet' if prefix | ipv4 else 'inet6' }} static
+iface {{ name }} {{ 'inet' if prefix | ipv4 else 'inet6' }} static
```

Comment:
why do you replace eth0 with {{ name }}? there are lots of places that assume the management port name is eth0. is this assumption broken on arista platform?

Reply:
this assumption is not broken.
1) Replace eth0 by '{{ name }}' as it should be
2) this fix allows to have more than one mgmt interfaces: eth0 + XXX + ...

Reply:
can you revert this?

I checked, current the name is Management0, I do not think we should use that.

Reply:
Sure, I'll remove this commit.